scripts/test-queue.sh

```
@@ -0,0 +1,388 @@
+fi
+
+# ============================================
+section "PUBLISH Command Tests"
+# ============================================
+
+# Test: Publish a simple message
+info "Test: queue publish simple message"
+PUBLISH_OUTPUT=$($CLI cloud queue publish "$QUEUE_NAME" '{"task":"test1"}' --json 2>&1) || true
+if echo "$PUBLISH_OUTPUT" | grep -q '"id".*"msg_'; then
+	pass "queue publish returns message ID"
+	MESSAGE_ID=$(echo "$PUBLISH_OUTPUT" | grep -o '"id":"msg_[^"]*"' | sed 's/"id":"//;s/"$//')
```

Shellcheck (0.11.0)

[warning] MESSAGE_ID appears unused. Verify use (or export if used externally).

(SC2034)